config/nav/config.json

```diff
@@ -47,7 +47,9 @@
 		"navStatusChannel": "/nav_status",
 		"repeaterDropInitChannel": "/rr_drop_init",
 		"repeaterDropCompleteChannel": "/rr_drop_complete",
-		"joystickChannel": "/autonomous"
+		"joystickChannel": "/autonomous",
+		"zedGimbalCommand": "/zed_gimbal_command",
```

Comment:
same fixes to channels as above, i guess on your end 'zedGimbalCommand' and 'zedGimbalPosition' are find since that is beyond my interface but it would make sense to change those to zedGimbalCmd and zedGimbalData

Reply:
zedGimbalCommand is just the config variable. Also, zedGimbalCmd doesn't exist anywhere so this change is actually the one defining the interface. Maybe I'm confused so please correct me where I'm wrong.

Reply:
ZedGimbalCmd the struct in the nav ICD where auton sends us the position it wants the zed to go to. zed_gimbal_cmd is what that channel should be called. it would make sense imo that your variable is zedGimbalCmd instead of zedGimbalCommand, because it'd match everything else, but that's just a personal opinion -- zedGimbalCmd vs zedGimbalCommand is beyond the esw interface

Reply:
This _does_ match everything else at the moment because there is nothing else yet.

Reply:
yeah im saying it makes to change them if the rest changes